xml/System.Buffers/IBufferWriter`1.xml

```diff
@@ -97,7 +108,20 @@
         <summary>Requests a span that is at least <paramref name="sizeHint" /> in size if possible.</summary>
         <returns>A span of size <paramref name="sizeHint" />, if it's available; otherwise, a memory span with the maximum available memory. If <paramref name="sizeHint" /> is 0, returns currently available memory.</returns>
         <remarks>To be added.</remarks>
+        <param name="sizeHint">The minimum length of the returned <see cref="T:System.Span`1" />. If 0, some, non-empty buffer is returned. </param>
```

Comment:
I think this is the problem @BillWagner. The description for the param should have been updated on line 107 instead of added here. Same for the returns and remarks that are also duplicated here.

Reply:
Thanks, @mairaw. I'd looked over this file multiple times and completely missed the duplication each time!